include/dictviewShim.hh

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
let's capitalize the file name

Reply:
In my testing, adding a shim does not work AT ALL. Let's delete this file and try another approach. That's why I put this PR in draft.